src/lib/Preview.js

```diff
@@ -959,7 +963,12 @@ class Preview extends EventEmitter {
                 this.reload(true); // Reload viewer without fetching updated file info from server
             }
         } catch (err) {
-            this.triggerError(err instanceof Error ? err : new Error(__('error_refresh')));
+            const error =
+                err instanceof PreviewError
+                    ? err
+                    : new PreviewError(ERROR_CODE.LOAD_VIEWER, __('error_refresh'), {}, err.message);
+
+            this.triggerError(error);
```

Comment:
This will show the `PreviewError` if that was thrown elsewhere, otherwise it constructs a new, more generic `PreviewError`